floodviz/static/js/hydrograph.js

```diff
@@ -14,44 +13,45 @@
  *
  */
 var hydromodule = function (options) {
-
 	var self = {};
 	var data_global = null;
 
+	var state = {};
+
 	var default_display_ids = null;
 	var timer = null;
 	var dblclick_armed = false;
 
-	var margin = {top: 60, right: 0, bottom: 30, left: 35};
-	var height = 500 * (options.height / options.width) - margin.top - margin.bottom;
+	var margin = {top: 60, right: 0, bottom: 30, left: 50};
```

Comment:
40 might work better for a left margin so it doesn't appear off-center with the text box